Maple2.Server.Game/Session/GameSession.cs

```diff
@@ -424,8 +424,8 @@ public sealed partial class GameSession : Core.Network.Session {
         Config.GatheringCounts.Clear();
         Send(UserEnvPacket.GatheringCounts(Config.GatheringCounts));
         // Death Counter
-        Config.DeathCounter = 0;
-        Send(RevivalPacket.Count(Config.DeathCounter));
+        Config.DeathCount = 0;
+        Send(RevivalPacket.Count(Config.DeathCount));
```

Comment:
if its always zero, we don't even need to pass the parameter

Reply:
This resets the death count on daily reset

Reply:
nvm I see what you mean